aep/general/0231/aep.md.j2

```diff
@@ -1,5 +1,182 @@
 # Batch methods: Get
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/42) for more
-information.
+Some APIs need to allow users to get a specific set of resources at a
+consistent time point (e.g. using a read transaction). A batch get method
+provides this functionality.
+
+## Guidance
+
+APIs **may** support batch get to retrieve a consistent set of resources.
+
+- The method's name **must** begin with `BatchGet`. The remainder of the method
+  name **must** be the plural form of the resource being retrieved.
+- The HTTP verb **must** be `GET`.
+- The HTTP URI **must** end with `:batchGet`.
+- The URI path **must** represent the collection for the resource, matching the
+  collection used for simple CRUD operations. If the operation spans parents, a
+  [wilcard](./reading-across-collections) **may** be accepted.
+- There **must not** be a request body.
+  - If a GET request contains a body, the body **must** be ignored, and **must
+    not** cause an error.
+
+### Request
+
+The request for a batch get method **should** be specified with the following
+pattern:
+
+- The request **must** include an array field which accepts the resource paths
+  specifying the resources to retrieve. The field **should** be named `paths`.
+  - If no resource paths are provided, the API **should** error with
+    `INVALID_ARGUMENT`.
+  - The parameter **must** be required.
+  - The parameter **should** identify the [resource type](./paths) that it
+    references.
+  - The parameter should define the pattern of the resource path values.
+  - The parameter should define the maximum number of paths allowed.
+- Other parameters besides `paths` **may** be "hoisted" from the [standard Get
+  request][get-request].
```

Comment:
```suggestion
```

I don't think this is explained very well - and I think removing unclear guidance is better than including it? we can re-add with a clearer description.

I don't see any documented additional parameters in the Get, so not sure if the guidance is even relevant: https://aep.dev/131/